Custom .c.o rule drops CFLAGS passed to configure; based on Debian
0002-Honor-CFLAGS-CPPFLAGS-passed-to-configure.patch

diff --git a/Makefile.am b/Makefile.am
index 82b8b21..468aa1f 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,8 +1,7 @@
 ## Process this file with automake to produce Makefile.in
 AUTOMAKE_OPTIONS = nostdinc
-ourdefs = -DPATH_RSSH_CONFIG=\"@sysconfdir@/rssh.conf\" -DPATH_CHROOT_HELPER=\"@libexecdir@/rssh_chroot_helper\"
-ourflags = @defcflags@ @static@ 
-AM_CFLAGS = $(ourflags)
+AM_CPPFLAGS = -DPATH_RSSH_CONFIG=\"@sysconfdir@/rssh.conf\" -DPATH_CHROOT_HELPER=\"@libexecdir@/rssh_chroot_helper\"
+AM_CFLAGS = @defcflags@ @static@
 bin_PROGRAMS = rssh
 libexec_PROGRAMS = rssh_chroot_helper
 nodist_rssh_SOURCES = main.c pathnames.h config.h
@@ -15,9 +14,6 @@ man_MANS = rssh.1 rssh.conf.5
 EXTRA_DIST = $(man_MANS) CHROOT SECURITY LICENSE rssh.spec mkchroot.sh conf_convert.sh
 base=`echo @PACKAGE_STRING@|tr " " "-"`
 
-.c.o:
-	$(CC) -c $(DEFS) $(ourdefs) $(AM_CFLAGS) $(CPPFLAGS) $(LDFLAGS) $<
-
 rpm:	dist
 	rpmbuild -ta --sign $(base).tar.gz
 	cp /users/ddm/tmp/RPM/i386/$(base)-1.i386.rpm .
